src/endpoint/s3/ops/s3_put_object.js

```diff
@@ -76,6 +79,11 @@ async function put_object(req, res) {
         };
     }
     res.setHeader('ETag', `"${reply.etag}"`);
+
+    if (reply.seq) {
+        res.seq = reply.seq;
```

Comment:
Seems like seq is only returned in namespace_nb flows. This will not work for other namespaces

Reply:
Right.
As we've discussed we can add a sequence to standalone, at the cost of adding a synchronized step to all s3 ops, so it's preferred for now to see if this is actually needed.

Reply:
But what about other namespaces (S3, Azure, FS) that are not in standalone?